packages/utils/package.json

```diff
@@ -37,22 +37,24 @@
   },
   "sideEffects": false,
   "dependencies": {
-    "@testing-library/react": "12.1.5",
-    "history": "4.10.1"
+    "history": "^4.10.1"
   },
   "devDependencies": {
-    "@edx/browserslist-config": "1.5.0",
-    "@edx/frontend-platform": "8.2.1",
-    "@openedx/frontend-build": "14.3.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-router-dom": "6.29.0",
-    "react-test-renderer": "17.0.2"
+    "@edx/browserslist-config": "^1.5.0",
+    "@edx/frontend-platform": "^8.2.1",
+    "@openedx/frontend-build": "^14.3.1",
+    "@testing-library/react": "^16.2.0",
+    "@testing-library/dom": "^10.4.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.29.0",
+    "react-test-renderer": "^18.3.1"
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
-    "react": "^16.12.0 || ^17.0.0",
-    "react-dom": "^16.12.0 || ^17.0.0",
+    "@testing-library/react": "^16.0.0",
```

Comment:
BREAKING CHANGE; library moved to `peerDep` vs `dep` to rely on the package installed by the MFE vs. installing it's own copy (reduce to one copy vs. keep around two).